mbuild/port.py

```diff
@@ -28,7 +34,7 @@ class Port(Compound):
         transform.
 
     """
-    def __init__(self, anchor=None, orientation=None, separation=0):
+    def __init__(self, anchor=None, orientation=[0, 1, 0], separation=0):
```

Comment:
This is a [mutable default arg](https://docs.quantifiedcode.com/python-anti-patterns/correctness/mutable_default_value_as_argument.html). Could you change this to:

```python
if orientation is None: 
    orientation = [0, 1, 0]
```

Reply:
Good call. That's how we had this originally, and I couldn't remember why, but this makes sense. I'll change it back.